src/advanced_inv.cpp

```diff
@@ -684,7 +706,7 @@ advanced_inv_listitem::advanced_inv_listitem( item *an_item, int index, int coun
         aim_location _area, bool from_veh )
     : idx( index )
     , area( _area )
-    , it( an_item )
+    , id(an_item->type->id)
```

Comment:
(Not tested, only theory) This looks like the `items` member is not initialized at all, callers seem to rely on the list containing at least one entry. Does that work?